<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>1427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер построения графиков</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../src.qrc">
    <normaloff>:/icons/favicon.png</normaloff>:/icons/favicon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="fileA_tab">
       <attribute name="title">
        <string>Файл А</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="fileA_tv"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fileB_tab">
       <attribute name="title">
        <string>Файл В</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="fileB_tv"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="fileC_tab">
       <attribute name="title">
        <string>Файл С</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="fileC_tv"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plot_tab">
       <attribute name="title">
        <string>График</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="plot_view" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="file_dock">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>516</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Файлы</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="fileDock_cntnt">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="FileWidget" name="fileA_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>140</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FileWidget" name="fileB_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>140</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="file_h_line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FileWidget" name="fileC_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>140</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="debug_btn">
       <property name="text">
        <string>Debug</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="file_v_spcr">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>486</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="plot_dock">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>505</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>График</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="plotDock_cntnt">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="AxisWidget" name="xAxis" native="true"/>
     </item>
     <item>
      <widget class="AxisWidget" name="yAxis" native="true"/>
     </item>
     <item>
      <widget class="AxisWidget" name="dxAxis" native="true"/>
     </item>
     <item>
      <widget class="AxisWidget" name="dyAxis" native="true"/>
     </item>
     <item>
      <widget class="QGroupBox" name="points_gb">
       <property name="title">
        <string>Точки</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="pointsSize_H_lyt">
          <item>
           <widget class="QLabel" name="pointSize_lbl">
            <property name="text">
             <string>Размер: </string>
            </property>
            <property name="buddy">
             <cstring>pointSize_H_sld</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="pointSize_H_sld">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pointSize_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pointsClr_H_lyt">
          <item>
           <widget class="ColorButton" name="pointClr" native="true">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="pointsClr_H_spcr">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="split_gb">
       <property name="title">
        <string>Разделяющая прямая</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="splitClr_H_lyt">
          <item>
           <widget class="ColorButton" name="splitClr" native="true">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="splitClr_H_spr">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="splitSize_H_lyt">
          <item>
           <widget class="QLabel" name="splitSize_lbl">
            <property name="text">
             <string>Размер: </string>
            </property>
            <property name="buddy">
             <cstring>splitSize_H_sldr</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="splitSize_H_sldr">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="splitSize_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="split_H_line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="angle_H_lyt">
          <item>
           <widget class="QLabel" name="angle_lbl">
            <property name="text">
             <string>Угол наклона (между прямой и осью абцисс):</string>
            </property>
            <property name="buddy">
             <cstring>angle_dial</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="angle_dial">
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="angle_sb">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="maximum">
             <number>365</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="delta_formLyt">
          <item row="0" column="0">
           <widget class="QLabel" name="hDelta_lbl">
            <property name="text">
             <string>Горизонтальный сдвиг:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>hDelta_H_sldr</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="hDelta_H_lyt">
            <item>
             <widget class="QSlider" name="hDelta_H_sldr">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="vDelta_lbl">
            <property name="text">
             <string>Вертикальный сдвиг:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>vDelta_H_sldr</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="vDelta_H_lyt">
            <item>
             <widget class="QSlider" name="vDelta_H_sldr">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="lineEq_gb">
          <property name="title">
           <string>Уравнение прямой:</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lineEq_y">
             <property name="text">
              <string>y = </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineEq_k">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>k</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineEq_x">
             <property name="text">
              <string>x</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineEq_plus">
             <property name="text">
              <string>+</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lineEq_b">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>b</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buld_lyt">
       <item>
        <spacer name="build_lh_spcr">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="build_btn">
         <property name="text">
          <string>Построить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="build_rh_spcr">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="plotDock_V_spcr">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Test mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>widgets/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileWidget</class>
   <extends>QWidget</extends>
   <header>widgets/filewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>widgets/colorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AxisWidget</class>
   <extends>QWidget</extends>
   <header>widgets/axiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../src.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pointSize_H_sld</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>pointSize_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>926</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>splitSize_H_sldr</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>splitSize_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>926</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angle_dial</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>angle_sb</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>889</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>926</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debug_btn</sender>
   <signal>clicked()</signal>
   <receiver>actionDebug</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
